build.sh

```diff
@@ -18,7 +18,41 @@ if [ ! -d "artifact" ]; then
     mkdir artifact
     chmod -R 755 artifact
 fi
-echo "For build number: ${build_number}"
 
 ## relocate the war file
 mv target/ctools-project-migration*.war artifact/ctools-project-migration.${build_number}.war
+
+function writeEnvironmentVariables {
+    # Values to be used while writing lines into the here document need to be evaluated here.
+    # Values written by the here document can be used when the Makefile itself is run.
+    # Variables values to be used when writting the script must be defined outside of the
+    # here document.  Values explicitly defined in the here document can only be used
+    # at build time, hence the escaping of the $ values.
+    local WEBAPPNAME_value=ctools-project-migration
+    local BUILD_value=${BUILD_NUMBER:-imaginary}
```

Comment:
one thing that I have never done in the build file is to surface the app server name in the /status page. Could that be done through the build file with a placeholder for the app server name?